examples/specs/interactive_bar_select_highlight.vl.json

```diff
@@ -18,8 +18,7 @@
   "mark": {
     "type": "bar",
     "fill": "#4C78A8",
-    "stroke": "black",
-    "cursor": "pointer"
```

Comment:
Assuming this was removed because the compiler now adds this automatically.

Reply:
Yep